fairscale/experimental/nn/sync_batchnorm.py

```diff
@@ -45,7 +46,7 @@ def _calculate_stats(input: Tensor, eps: float, process_group: ProcessGroup) ->
     return mean, var, invstd, total_count
 
 
-if torch.__version__.split(".")[:2] >= ["1", "7"]:
+if torch_version() >= [1, 7]:
```

Comment:
there is no logically change in this PR?

Reply:
There is for torch 1.10.0.

```
["1", "10"] < ["1", "7"]
```
```
[1, 10] > [1, 7]
```

Reply:
Gosh, so subtle! Looks great!